gapis/server/server.go

```diff
@@ -198,6 +198,18 @@ func (s *server) LoadCapture(ctx context.Context, path string) (*path.Capture, e
 	return p, nil
 }
 
+func (s *server) SaveCapture(ctx context.Context, c *path.Capture, path string) error {
+	ctx = log.Enter(ctx, "SaveCapture")
+	if !s.enableLocalFiles {
+		return fmt.Errorf("Server not configured to allow writing of local files")
+	}
+	buf := bytes.Buffer{}
```

Comment:
You can avoid a temporary copy of the capture here:

```go

	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY, 0666)
	if err != nil {
		return err
	}
	defer f.Close()
	if err := capture.Export(ctx, c, &buf); err != nil {
		return err
	}
```

Reply:
Done